functional/basic-attestation-without-mtls/test.sh

```diff
@@ -0,0 +1,147 @@
+            export TPM2TOOLS_TCTI=tabrmd:bus_name=com.intel.tss2.Tabrmd
+            rlRun "limeInstallIMAConfig"
+            rlRun "limeStartIMAEmulator"
+        else
+            rlServiceStart tpm2-abrmd
+        fi
+        sleep 5
+        # start keylime_verifier
+        rlRun "limeStartVerifier"
+        rlRun "limeWaitForVerifier"
+        rlRun "limeStartRegistrar"
+        rlRun "limeWaitForRegistrar"
+        # when using unit files we need to adjust them
+        if [ -f /usr/lib/systemd/system/keylime_agent.service -o -f /etc/systemd/system/keylime_agent.service ]; then
+            rlRun "mkdir -p /etc/systemd/system/keylime_agent.service.d/"
+            rlRun "cat > /etc/systemd/system/keylime_agent.service.d/20-keylime_dir.conf <<_EOF
+[Service]
+Environment=\"TCTI=${TPM2TOOLS_TCTI}\"
+Environment=\"RUST_LOG=keylime_agent=trace\"
+_EOF"
+            rlRun "systemctl daemon-reload"
+        fi
+        # create allowlist and excludelist
+        limeCreateTestLists
+        # create expect script to add agent
+        rlRun "cat > add.expect <<_EOF
+set timeout 20
+spawn lime_keylime_tenant -v 127.0.0.1 -t 127.0.0.1 -u $AGENT_ID --allowlist allowlist.txt --exclude excludelist.txt --include payload --cert default -c add
+expect \"Please enter the password to decrypt your keystore:\"
+send \"keylime\n\"
+expect eof
+_EOF"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Check that agent cannot start without explicitly enabling insecure payload"
+        if [ -f /usr/lib/systemd/system/keylime_agent.service -o -f /etc/systemd/system/keylime_agent.service ]; then
+            rlRun "limeStartAgent"
+        else
+            rlRun "TCTI=${TPM2TOOLS_TCTI} RUST_LOG=keylime_agent=trace limeStartAgent"
+        fi
+        rlRun "limeWaitForAgentRegistration ${AGENT_ID}" 1
+        rlAssertGrep "enable_insecure_payload. has to be set to .True." $(limeAgentLogfile) -E
+    rlPhaseEnd
+
+    rlPhaseStartTest "Check that empty script_payload allows the agent to start"
+        rlRun "limeUpdateConf cloud_agent payload_script ''"
+        if [ -f /usr/lib/systemd/system/keylime_agent.service -o -f /etc/systemd/system/keylime_agent.service ]; then
+            rlRun "systemctl daemon-reload"
+            rlRun "limeStartAgent"
+        else
+            rlRun "TCTI=${TPM2TOOLS_TCTI} RUST_LOG=keylime_agent=trace limeStartAgent"
+        fi
+        rlRun "limeWaitForAgentRegistration ${AGENT_ID}"
+        rlRun "expect add.expect"
+        rlRun "limeWaitForAgentStatus $AGENT_ID 'Get Quote'"
+        rlRun -s "lime_keylime_tenant -c cvlist"
+        rlAssertGrep "{'code': 200, 'status': 'Success', 'results': {'uuids':.*'$AGENT_ID'" $rlRun_LOG -E
+        rlAssertGrep "mTLS disabled" $(limeAgentLogfile)
+        rlAssertGrep "payloads cannot be deployed" $(limeAgentLogfile)
+    rlPhaseEnd
+
+    rlPhaseStartTest "Enable insecure payload and set script_payload"
+        rlRun -s "lime_keylime_tenant -v 127.0.0.1 -t 127.0.0.1 -u $AGENT_ID -c delete"
```

Comment:
Adding `lime_keylime_tenant -r 127.0.0.1 -t 127.0.0.1 -u d432fbb3-d2f1-4a97-9ef7-75bd81c00000 -c regdelete` below fixes the CI failure.

Reply:
Rebased on top of current `main` and added this.